modules/@apostrophecms/piece-type/index.js

```diff
@@ -96,6 +96,13 @@ module.exports = {
       }
     }
   },
+  moreMenu: {
+    add: {
+      import: {
```

Comment:
Label this `// TEMP` so we remember to remove it in the import work.

Reply:
Yes done